.github/workflows/mobile-android_build.yml

```diff
@@ -68,10 +68,16 @@ jobs:
         java-package: jdk
         architecture: x64
         distribution: zulu
-    - run: |
+    - name: 'Install dependencies'
+      run: |
         cd mobile
-        ./ci/mac_ci_setup.sh --android
-      name: 'Install dependencies'
+        ./ci/linux_ci_setup.sh
+    # https://github.blog/changelog/2023-02-23-hardware-accelerated-android-virtualization-on-actions-windows-and-linux-larger-hosted-runners/
+    - name: Enable KVM group permissions
```

Comment:
i think this and the adb install stuff belong in a github action

i can follow up to add and shift the code

Reply:
Thanks.

Reply:
I totally trust your judgement here, @phlax. Can you explain more about what makes sense in the workflow and what make sense in an action? I have too little experience to have a clue :)

Reply:
well, kinda forming strategies as we speak - but probs we dont want any bash etc in workflows - split those out to actions for reusablitiy and isolation and use workflows for controlling _workflows_

Reply:
Ah, that makes sense.